src/FormField/AutoComplete.php

```diff
@@ -157,4 +156,59 @@ public function getInput()
             'value' => $this->getValue(),
         ]);
     }
+
+    /**
+     * Set Semantic-ui dropdown module settings.
+     *
+     * For example, using this setting will automatically submit
+     * form when field value is changes.
+     * $ac->setDropdownSettings(['allowReselection' => true,
+     *                           'selectOnKeydown' => false,
+     *                           'onChange'        => new atk4\ui\jsExpression('function(value,t,c){
+     *                                                          if ($(this).data("value") !== value) {
+     *                                                            $(this).parents(".form").form("submit");
+     *                                                            $(this).data("value", value);
+     *                                                          }
+     *                                                         }'),
+     *                          ]);
+     *
+     * @param array $settings
+     *
+     * @return $this
+     */
+    public function setDropdownSettings($settings)
```

Comment:
we don't generally use setters in Agile UI unless they do more than assign a variable, so lets follow that pattern.